generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -203,8 +203,10 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
         , <%= rel.relationshipFieldNamePlural %>: mapIdList(values.<%= rel.relationshipFieldNamePlural %>)
         <%_ }) _%>
       }
-      <%_ if (uniqueRelationFields.has('users')) { _%>
-      entity.user = values.user;
+      <%_ if (uniqueRelationFields.has('users') && authenticationType === 'oauth2' && applicationType === 'gateway' && dto !== 'mapstruct') { _%>
```

Comment:
Okay I'm lost. Why only a gateway app?

Reply:
It's only necessary to send a valid `User` (which requires both the `login` and `id` fields) for OAuth2 Gateways because OAuth2 Microservice entities can have relationships with `User` 

This is not possible with JWT microservices.  For monoliths, only the `id` is needed.